omero/sysadmins/unix/server-centos6-ius.txt

```diff
@@ -0,0 +1,212 @@
+
+Install Java::
+
+    yum install java-1.8.0-openjdk
+
+Install and start PostgreSQL::
+
+    yum install http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-redhat94-9.4-1.noarch.rpm 
+    yum install postgresql94-server postgresql94-contrib
+    service postgresql-9.4 initdb
+    chkconfig postgresql-9.4 on
+    service postgresql-9.4 start
+
+For the steps described above: 
+
+	link to be added
+
+Setting up the OMERO.server
+---------------------------
+This is an example walkthrough for installing OMERO on Centos6 with Python 2.7 using
+a dedicated system user.
+You can use this as a guide for setting up your own test server. For production use you should also read the pages listed under :ref:`index-optimizing-server`.
+
+For convenience in this walkthrough the main OMERO configuration options have
+been defined as environment variables. When following this walkthrough you can
+either use your own values, or alternatively source the following file:
+
+.. literalinclude:: walkthrough/settings.env
+   :start-after: Substitute
+
+:download:`settings.env <walkthrough/settings.env>`
+
+Once Python2.7, Ice3.5 and the additional packages are installed:
+
+Create an omero system user, and a directory for the OMERO repository:
+
+.. literalinclude:: walkthrough/step02_all_setup.sh
+
+:download:`step02_all_setup.sh <walkthrough/step02_all_setup.sh>`
+
+
+Create a database user and initialize a new database for OMERO:
+
+.. literalinclude:: walkthrough/step03_all_postgres.sh
+
+:download:`step03_all_postgres.sh <walkthrough/step03_all_postgres.sh>`
+
+
+To use Nginx, as **root**, run::
+	
+	yum install nginx
+
+To use Apache 2.4, as **root**, run::
+	
+	yum install httpd24u
+
+To install the remaining dependencies, create a virtual environment::
+
+	virtualenv -p /usr/bin/python2.7 /home/omero/omeroenv
+	source /home/omero/omeroenv/bin/activate
+	#upgrade pip
+	/home/omero/omeroenv/bin/pip install --upgrade pip
+
+The following settings will need adding to your OMERO startup script
+or to the OMERO user's environment (for example in a shell startup
+script). Add the absolute path to the :file:`bin` directory of 
+the virtual environment :file:`/home/omero/omeroenv` to the ``PATH``
+variable::
+
+    ICE_HOME=/opt/Ice-3.5.1
+    PATH="${ICE_HOME}/bin:/home/omero/omeroenv/bin:$PATH"
+    export LD_LIBRARY_PATH="${ICE_HOME}/lib64:${ICE_HOME}/lib:$LD_LIBRARY_PATH"
+    export PYTHONPATH="${ICE_HOME}/python:$PYTHONPATH"
+    export SLICEPATH="${ICE_HOME}/slice"
+
+These settings will enable Python 2.7, and set the necessary
+environment variables for Ice 3.5 to work.	
+
+Install additional Python modules::
+
+	/home/omero/omeroenv/bin/pip2.7 install pillow numpy matplotlib
+
+Install Django::
+
+	/home/omero/omeroenv/bin/pip2.7 install "Django>=1.8,<1.9"
+
+To use Nginx, install Gunicorn::
+
+	/home/omero/omeroenv/bin/pip2.7 install gunicorn
+
+To use Apache 2.4, install mod_wsgi::
+
+	/home/omero/omeroenv/bin/pip2.7 install mod_wsgi-httpd
```

Comment:
@aleksandra-tarkowska: I read that too
if we do not want to pip install mod_wsgi-httpd, we need to install more globally
Since we only install httpd24 . you will have the following error i.e. ``The 'apxs' command appears not to be installed`` if ``mod_wsgi-httpd`` is not installed
I went for the option isolating everything in the virtualenv.